### PR TITLE
Fix multiple serialized values

### LINKED ISSUE #12803: Multipe serialized configurations in the database
There were stange configs in the database that leads to Error 500

It was like
```
[punkrock@libra html]$ php bin/console.php config system url
system.url => s:48:"s:40:"s:32:"https://punkrock-underground.com";";";
```

See full description in this thread: https://anonsys.net/display/bf69967c-8963-ebd5-b759-d94656945143

### PR DESCRIPTION
**Pull request by nupplaphil**
Closes #12803

### CODE REVIEW

**Review by Quix0r — APPROVED** [review]
Looking fine to me.

File: src/Core/Config/Util/SerializeUtil.php [hunk 1]
```diff
@@ -28,10 +28,24 @@
  */
 class SerializeUtil
 {
+	/**
+	 * Checks if the value needs to get unserialized and returns the unserialized value
+	 *
+	 * @param mixed $value A possible serialized value
```
Comment (MrPetovan) [inline src/Core/Config/Util/SerializeUtil.php]:
```suggestion
	 * @param mixed $value A possibly serialized value
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Language

**Review by MrPetovan — APPROVED** [review]

**Review by Quix0r — APPROVED** [review]

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Database version has increased since your changes, please rebase your branch.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**